python/eos/analysis.py

```diff
@@ -377,6 +377,7 @@ def sample_pmc(self, log_proposal, step_N=1000, steps=10, final_N=5000, rng=np.r
         :param steps: Number of adaptation steps.
         :param final_N: Number of samples that shall be drawn after all adaptation steps.
         :param rng: Optional random number generator (must be compatible with the requirements of pypmc.sampler.importance_sampler.ImportancSampler)
+        :param return_final_only: If set to True, only returns the samples and weights of the final adaptation step.
```

Comment:
The final step is a pure evaluation step without adaptations, right? So this is slightly off.

How about:
``If set to True, only returns the samples and weights of the final sampling step, after all adaptations have finished.`` ?